### PR TITLE
Docs Fixes

### PR DESCRIPTION
**Pull request by sharadregoti**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

___

### **PR Type**
Documentation

___

### **Description**
- Corrected terminology in the documentation from "rate limit" to "size limit" for API-level configuration in the Tyk OAS APIs.
- Clarified the description of the issue related to the ordering of Tyk OAS API paths in the release notes.
- Removed an unnecessary comment regarding the formatting of release notes.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>request-size-limit-tyk-oas.md</strong><dd><code>Correct terminology for API-level size limit</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-gateway/middleware/request-size-limit-tyk-oas.md

<li>Corrected terminology from "rate limit" to "size limit" for API-level <br>configuration.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5735/files#diff-3565a648321c9e7a85ed1386d9e12a0ccbe1bfcca5583ca1948db312f329ec72">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>version-5.3.md</strong><dd><code>Clarify OAS API path ordering and remove comment</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md

<li>Clarified the description of OAS API path ordering issue.<br> <li> Removed unnecessary comment about release notes formatting.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5735/files#diff-ead598197a92e18014f4f7c35c341621948fb6cdf66d45c1dd05de21bc2b35aa">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-gateway/middleware/request-size-limit-tyk-oas.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ There are three different levels of granularity that can be used when configurin
 
 ### Applying a size limit for a specific API
 
-The API-level rate limit has not yet been implemented for Tyk OAS APIs.
+The API-level size limit has not yet been implemented for Tyk OAS APIs.
 
-You can work around this by implementing a combination of endpoint-level rate limits and [allow]({{< ref "product-stack/tyk-gateway/middleware/allow-list-tyk-oas#configuring-the-allow-list-in-the-tyk-oas-api-definition" >}}) or [block]({{< ref "product-stack/tyk-gateway/middleware/block-list-tyk-oas#configuring-the-block-list-in-the-api-designer" >}}) lists.
+You can work around this by implementing a combination of endpoint-level size limits and [allow]({{< ref "product-stack/tyk-gateway/middleware/allow-list-tyk-oas#configuring-the-allow-list-in-the-tyk-oas-api-definition" >}}) or [block]({{< ref "product-stack/tyk-gateway/middleware/block-list-tyk-oas#configuring-the-block-list-in-the-api-designer" >}}) lists.
```
Comment (sharadregoti) [inline tyk-docs/content/product-stack/tyk-gateway/middleware/request-size-limit-tyk-oas.md:25]: @andyo-tyk Can you help me out with reference URLs? That need to be changed.

Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-gateway/middleware/request-size-limit-tyk-oas.md:25]: The references are fine as they are. :)

**Review by andyo-tyk — APPROVED** [review]
LGTM

**Pull request merged**